### ISSUE TITLE
Internal Error (parameter-list-wrapping)

### ISSUE THREAD

**Issue by SUPERCILEX** [shyiko/ktlint#164]
Functions like this seem to cause the issue:
```kt
fun DialogFragment.show(
        manager: FragmentManager,
        tag: String,
        args: Bundle = Bundle(),
        argsListener: (Bundle.() -> Unit)? = null
) {
    arguments = args.apply { argsListener?.invoke(this) }
    show(manager, tag)
}
```

**Comment by shyiko** [comment]
Hi Alex. Can you please provide a complete example? (I'm unable to reproduce)

**Comment by SUPERCILEX** [comment]
If you [update ktlint to `0.16.1`](https://github.com/SUPERCILEX/Robot-Scouter/blob/0d1b2883101051d7aaa3ca667753dd5d37246179/quality/quality.gradle#L12) on [my repo](https://github.com/SUPERCILEX/Robot-Scouter), you'll get a ton of these issues.

**Comment by shyiko** [comment]
Fixed in [0.17.0](https://github.com/shyiko/ktlint/releases/tag/0.17.0) (Maven Central sync might take up to 25 minutes).

**Comment by SUPERCILEX** [comment]
Awesome, thanks!
